.ibm/pipelines/resources/config_map/app-config-rhdh.yaml

```diff
@@ -17,6 +17,148 @@ app:
 dynamicPlugins:
   rootDirectory: dynamic-plugins-root
   frontend:
+    red-hat-developer-hub.backstage-plugin-dynamic-home-page:
```

Comment:
Can this entire customization configuration be moved to a different location, such as another YAML file? The app-config.yaml file is becoming too large.

Reply:
Created a new config file called dynamic-homepage-and-sidebar-config.yaml